src/partials/section-gallery.html

```diff
@@ -0,0 +1,98 @@
+<div class="section-gallery">
+  <div class="container">
+    <div class="left-section">
```

Comment:
Aby ustawić dwie kolumny użyłeś flexa. Oczywiście jest to wykonanie okej, ale skoro mamy juz w projekcie bootstrapa i wiele razy używaliśmy go do ustawiania podstawowego ukłądu kolumn, to czy nie lepiej również tutaj to zrobić?

Chodzi mi tu o ten podstawowy podział. W innych miejsach (jak ul) flex ma juz sens.